### ISSUE TITLE
How could I get a softmax output of these models?

### ISSUE THREAD

**Issue by liqin-z** [qubvel/classification_models#49]
In this case i don't want the prediction class so that I add `include_top = False` inside the model. I want a softmax layer at the end of the network, and get the output of specified size. Any code to help please?

**Comment by qubvel** [comment]
Did you read README? https://github.com/qubvel/classification_models/blob/master/README.md#model-fine-tuning-example

**Comment by graveszhang** [comment]
BTW, this repo is so
